incubator/hnc/config/oldDefaultExternalCert/kustomization.yaml

```diff
@@ -0,0 +1,67 @@
+# Adds namespace to all resources.
```

Comment:
Rather than duplicate all these files, for now I'd just comment out the relevant parts in the main one. That's what kubebuilder gives us in the first place.